test/unit/output/ferc1_test.py

```diff
@@ -0,0 +1,119 @@
+"""Tests for the FERC Form 1 output functions.
+
+Calculation tree generation:
+- [x] Test base recusion case (leaf node w/ empty calculations)
+- [x] Test simple 1-level recursion w/ 1-2 child nodes.
+- [x] Test calculation with non-unique source table (should fail)
+- Test calculation that references an outside table (???)
```

Comment:
Is this the `KeyError` test case down below now?

Reply:
Ah yeah -- but I think @cmgosnell and I worked out what should actually be happening, which is that we should never encounter this situation because `ExplodeMeta` has already revised the calculation metadata to remain within the boundaries of the Exploded Metadata.  So I'll change this to an `xfail`.